src/TypeLevel/DataSchema.purs

```diff
@@ -0,0 +1,238 @@
+module TypeLevel.DataSchema
+  ( PSchema
+  , class HasPlutusSchema
+  , PNil
+  , PCons
+  , ApPCons
+  , Id
+  , I
+  , type (:+)
+  , IxK
+  , MkIxK
+  , MkIxK_
+  , type (@@)
+  , Field
+  , MkField
+  , MkField_
+  -- , NoRec
```

Comment:
Why is this commented out?

Reply:
Dead code (predates the fancy syntactic sugar in DataSchema). Will remove.